src/default-regex.ts

```diff
@@ -0,0 +1,7 @@
+// These rules should match the rules as specified in
+// https://github.com/openfga/api/blob/main/openfga/v1/openfga.proto
```

Comment:
Can you link to the appropriate part please? 🙏🏽

Reply:
The problem is that we either have to link with respect to a specific version or the line numbers will become out of date